src/call_builder.ts

```diff
@@ -131,7 +131,26 @@ export class CallBuilder<
       createTimeout();
 
       if (es) {
+        // when receiving the close message from Horizon we should
+        // close the connection and recreate the event source
+        let closed = false;
+        const onClose = () => {
+          if (closed) {
```

Comment:
`onClose()` should only ever be invoked once but I added a `closed` flag to be defensive